src/access_nri_intake/utils.py

```diff
@@ -102,7 +102,21 @@ def validate_against_schema(instance: dict, schema: dict) -> None:
         Validator, type_checker=type_checker
     )
 
-    TupleAllowingValidator(schema).validate(instance)
+    issues = list(TupleAllowingValidator(schema).iter_errors(instance))
+
+    if len(issues) > 0:
+        issue_str = ""
+        for i, issue in enumerate(issues):
```

Comment:
passing enumerate a `start=1` argument would remove the need for the `i+1` on lines 112, 115

Reply:
Done